bitnami/rabbitmq/templates/NOTES.txt

```diff
@@ -39,7 +39,7 @@ Note that the credentials are saved in persistent volume claims and will not be
 This is applicable when no passwords are set and therefore the random password is autogenerated. In case of using a fixed password, you should specify it when upgrading.
 More information about the credentials may be found at https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues/#credential-errors-while-upgrading-chart-releases.
 
-RabbitMQ can be accessed within the cluster on port {{ $serviceNodePort }} at {{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
+RabbitMQ can be accessed within the cluster on port {{ $servicePort }} at {{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}
```

Comment:
Could you please remove the `serviceNodePort` variable definition? It was only used here.